### ISSUE TITLE
[Android] Release mode is calling for .bundle instead of .jsbundle

### ISSUE THREAD

**Issue by joaom182** [joltup/react-native-threads#42]
```
Crash java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.RuntimeException: Unable to load script from assets 'threads/pre-load-data-worker.thread.bundle'. Make sure your bundle is packaged correctly or you're running a packager server.
```

![screen shot 2018-07-28 at 11 29 18](https://user-images.githubusercontent.com/3612098/43357535-8379af28-9259-11e8-9c5a-9602ab1f708a.png)

**Comment by Traviskn** [comment]
You're right! The issue is here, we need up update the code to look for a `.jsbundle` suffix rather than a `.bundle` suffix

**Comment by Traviskn** [comment]
yep that's the only change. I found that in our app we're just generating a `.bundle` file for android and a `.jsbundle` for iOS, but it would definitely be nice to make that more consistent

**Comment by saalihou** [comment]
This and the fact that the README states that the bundle should be generated with a `.jsbundle` is probably confusing a lot of newcomers who wouldn't have the reflex to look at the logcat. Is there a reason it is still not fixed (backwards compatibility ?) or is it just noone did a PR.
